FastSimulation/SimplifiedGeometryPropagator/interface/FastTrackDeDxProducer.h

```diff
@@ -0,0 +1,99 @@
+#ifndef FASTSIM_TRACKRECODEDX_FASTTRACKDEDXPRODUCER_H
+#define FASTSIM_TRACKRECODEDX_FASTTRACKDEDXPRODUCER_H
```

Comment:
The contents of this header would actually be better moved to the `.cc` file.

Reply:
perhaps you're right. the important thing to me is to add a single member and setter to this existing class. if you suggest which content should be moved, I can try to do it.

Reply:
I tried moving some of the material, but hopefully it's ok to leave them as is for the moment.